tests/ssl-opt.sh

```diff
@@ -525,6 +525,46 @@ check_server_hello_time() {
     fi
 }
 
+# Get handshake memory usage from server or client output and check if this value 
+# is not higher then maximum given in the argument
+handshake_memory_check() {
+    MAX_MEMORY="$1"
+    OUTPUT_FILE="$2"
+
+    # Get memory usage from patern like "Heap memory usage after handshake: 23112 bytes. In peak was 33112"
+    MEMORY_USAGE=$(sed -n 's/.*Heap memory usage after handshake: //p' < "$OUTPUT_FILE" | grep -o "[0-9]*" | head -1)
+	
+    # Check if memory usage was read
+    if [ -z "$MEMORY_USAGE" ]; then
+        return 1
+    fi
+
+    # Check if memory usage is below max value
+    if [ "$MEMORY_USAGE" -gt "$MAX_MEMORY" ]; then
+        return 1
+    else
+        return 0
+    fi
+}
+
+# Get handshake memory usage from server or client output and put it into variable named
+# in the argument 
+handshake_memory_get() {
+    OUTPUT_VARIABLE="$1"
+    OUTPUT_FILE="$2"
+
+    # Get memory usage from patern like "Heap memory usage after handshake: 23112 bytes. In peak was 33112"
```

Comment:
This needs updating, same as the other similar comment.

Reply:
done